configs/coderello_docs.json

```diff
@@ -0,0 +1,26 @@
+{
+  "index_name": "coderello_docs",
+  "start_urls": [],
+  "stop_urls": [],
+  "sitemap_urls": [
+    "https://coderello.com/sitemap.xml"
+  ],
+  "selectors": {
+    "lvl0": ".docsearch-content h1",
+    "lvl1": ".docsearch-content h2",
+    "lvl2": ".docsearch-content h3",
+    "lvl3": ".docsearch-content h4",
+    "lvl4": ".docsearch-content h5",
+    "lvl5": ".docsearch-content h6",
+    "text": ".docsearch-content p, .docsearch-content li"
+  },
+  "custom_settings": {
+    "attributesForFaceting": ["language", "version", "package", "page"]
```

Comment:
Did you make sure to have 4 DocSearch meta tags in every page?

Reply:
I have these tags on every documentation page, yeah.

Reply:
For example:

```
<meta name="docsearch:language" content="en">
<meta name="docsearch:version" content="1.0">
<meta name="docsearch:package" content="laravel-nova-lang">
<meta name="docsearch:page" content="introduction">
```

For this page: https://coderello.com/docs/laravel-nova-lang/1.0/introduction

Reply:
However, the main page of the website doesn't have ones, because it should not be indexed.
So it doesn't have either the root class `.docsearch-content` and `docsearch:*` meta tags.

Reply:
I'm also planning to add a blog to this website. But it should not be indexed because blog ≠ docs 🙂
How could we handle it? Will the excess of the root class and meta tags command the parser to skip these pages?

Reply:
*not excess, but absense

Reply:
We do not usually index blog section.  However we might still encompass this part of your documentation if it does help the community with enlightening content. If so, please create a dedicated start_url item with a dedicated `selector_key`.

Please note that meta tags are retrieved on every page despite of the selector_key